megamek/src/megamek/common/Mech.java

```diff
@@ -7082,6 +7159,100 @@ public boolean addDualCockpit() {
         setCockpitType(COCKPIT_DUAL);
         return true;
     }
+	
+    public boolean addSuperheavyIndustrialCockpit() {
+        if (getEmptyCriticals(LOC_HEAD) < 5) {
+            return false;
+        }
+        addCritical(LOC_HEAD, 0, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_LIFE_SUPPORT));
+        addCritical(LOC_HEAD, 1, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_SENSORS));
+        addCritical(LOC_HEAD, 2, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_COCKPIT));
+        addCritical(LOC_HEAD, 4, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
```

Comment:
Should this be 3?

Reply:
I just copy-paste it from normal industrial cockpit and it would be correct - or at least it follows the current setting. On MegaMekLab, for normal cockpit the location 3(the fourth location from the top) is empty, and for smalls 4 and 5(fifth and sixth) are empty.